unittests/test_flush_auditlog.py

```diff
@@ -0,0 +1,45 @@
+from dojo.tasks import flush_auditlog
+from .dojo_test_case import DojoTestCase
+from django.conf import settings
```

Comment:
```suggestion
from django.test import override_settings
```
This is not needed anymore.
But you will need this :)